### PR TITLE
add check for read only filesystem

### PR DESCRIPTION
**Pull request by jendap**

### CODE REVIEW

File: tensorflow/tools/ci_build/builds/with_the_same_user [hunk 1]
```diff
@@ -27,6 +27,15 @@ set -e
 
 COMMAND=("$@")
 
+if ! touch /this_is_writable_file_system; then
+  echo "You can't write to your filesystem!"
+  echo "If you are in Docker you should check you do not have too many images" \
+      "with too many files in them. Docker have some issue with it."
```
Comment (caisq) [inline tensorflow/tools/ci_build/builds/with_the_same_user]: nit: have --> has

**Comment by caisq** [comment]
@jendap Do you plan to submit the same changes internally as well? The PR will not affect CL tests until that is done or someone pulls latest changes to internal.

**Comment by jendap** [comment]
I forgot about this one. Sorry. It is updated now.

@caisq it is not time sensitive. It will get synced one day ;-)

**Comment by vrv** [comment]
ok to merge?

**Pull request merged**